packages/README.md

```diff
@@ -4,10 +4,11 @@ This repository uses [npm workspaces](https://docs.npmjs.com/cli/v10/using-npm/w
 
 ## Creating a New Package
 
-When creating a new package, you need to provide at least the following:
+When creating a new package, you need to provide at least the following. Packages bundled in Gutenberg or WordPress must include a `wpScript` and or `wpScriptModuleExports` field in their `package.json` file. See the details below.
 
 1. `package.json` based on the template:
-    ```json
+
+    ```jsonc
```

Comment:
Should this just be `json`?

Otherwise, changes look good.

Reply:
I've added some comments. Those aren't allowed in "strict" JSON, `jsonc` will highlight it correctly.